bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/types/BAnnotatableType.java

```diff
@@ -41,6 +41,10 @@ public void setAnnotations(BMap<BString, Object> annotations) {
         this.annotations = annotations;
     }
 
+    public BMap<BString, Object> getAnnotations() {
+        return this.annotations;
```

Comment:
Shall we provide copy of the map. external user can change the map if we provide the original reference.

Reply:
@warunalakshitha We seem to have a setter for `annotations` in `BAnnotatableType` already. So do we really need to avoid modifying the annotations in here? And maybe we can remove `getAnnotations` from `AnnotatableType` interface  instead. WDYT?

Reply:
Changed in a365752